docs/domains/Domain.md

```diff
@@ -98,12 +98,13 @@ An element representing a cluster in the domain configuration.
 | `overrideDistributionStrategy` | string | Determines how updated configuration overrides are distributed to already running WebLogic servers following introspection when the domainHomeSourceType is PersistentVolume or Image.  Configuration overrides are generated during introspection from secrets, the overrideConfigMap field, and WebLogic domain topology. Legal values are DYNAMIC (the default) and ON_RESTART. See also introspectVersion. |
 | `overridesConfigMap` | string | The name of the config map for WebLogic configuration overrides. If this field is specified it overrides the value of spec.configOverrides. |
 | `secrets` | array of string | A list of names of the secrets for WebLogic configuration overrides or model. If this field is specified it overrides the value of spec.configOverrideSecrets. |
+| `istio` | [Istio](#istio) | Moved under configuration. Should we remove this ??? |
 
 ### Experimental
 
 | Name | Type | Description |
 | --- | --- | --- |
-| `istio` | [Istio](#istio) | Istio service mesh integration configuration. |
+| `istio` | [Istio](#istio) | Moved under configuration. Should we remove this ??? |
```

Comment:
Yes, per Mark's response, let's go ahead and remove experimental.

Reply:
removed